core/src/OpenACC/Kokkos_OpenACCSpace.cpp

```diff
@@ -80,24 +81,8 @@ void *Kokkos::Experimental::OpenACCSpace::impl_allocate(
     const Kokkos::Experimental::OpenACC &exec_space, const char *arg_label,
     const size_t arg_alloc_size, const size_t arg_logical_size,
     const Kokkos::Tools::SpaceHandle arg_handle) const {
-  static_assert(sizeof(void *) == sizeof(uintptr_t),
-                "Error sizeof(void*) != sizeof(uintptr_t)");
-
-  void *ptr = nullptr;
-
-  // FIXME_OPENACC multiple device instances are not yet supported, and thus
-  // exec_space is ignored for now.
   (void)exec_space;
```

Comment:
Why do we need this?

Reply:
Silent warnings.  The other way to do it is not name the argument.